src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -130,19 +133,41 @@ export const TablesUserList: React.FC<TablesUserListProps> = ({
     [recentVenueUsers, user, venueName, videoRoom]
   );
 
+  const itemsToObjectByRefReducer = useCallback(
+    (obj: Record<string, WithId<User>[]>, table: Table) => ({
```

Comment:
`obj`-> `acc` ? (accumulator)

Reply:
`obj` is probably fine here given the name of the reducer also calls it 'object'. I try and aim to name it in a way that it would make the most sense to a human.

Reply:
left as `obj`